Tests/SpotifyWebAPIMainTests/API Tests/SpoifyAPIArtistTests.swift

```diff
@@ -788,3 +809,149 @@ final class SpotifyAPIAuthorizationCodeFlowPKCEArtistTests:
     func testRelatedArtists() { relatedArtists() }
     
 }
+
+// MARK: - Proxy -
+
+final class SpotifyAPIClientCredentialsFlowProxyArtistTests:
```

Comment:
These sound like useful tests; however they also can be singled out and added to master without needing to be part of this PR.

Reply:
In case you'll be asking this later: Yes, it is likely there will be one or two lines of code conflict from master and tests here. However, better to address this small divergence with a single LOC change, than merge it in as part of the same PR.

Reply:
No, these tests, and all other tests with the word `Proxy` in the name are specific to the new authorization backend managers. See the `SpotifyAPIClientCredentialsFlowProxyTests` base class. In order to understand the tests, you must carefully review the protocols they conform to and the base classes they inherit from. `SpotifyAPIClientCredentialsFlowProxyArtistTests` runs the same tests as `SpotifyAPIClientCredentialsFlowArtistTests`, except using the `ClientCredentialsFlowProxyBackend`.